### DISCUSSION TITLE
Metal Business Card Cutout and Engrave

### DISCUSSION THREAD

**Discussion by chrisglitz** [arkypita/LaserGRBL#1861] [Q&A]
Hello, I'm looking to create a business card using an Atomstack A5 Pro, a 0.45mm Metal Business Card Blank measuring 3.4 X 2.13 in, and Laser GRBL. 

On the card, I'd like to make the logo cutout and text of the card engraved.

Is it possible to do this in one pass? Any suggestions on how to accomplish this?

Thanks

**Comment by Tribo-2dot0** [comment]
Use the Append File function in the File menu.
First the engraving file, then append the vector.

**Comment by chrisglitz** [comment]
I’ll  give it a shot, thanks!
